### DISCUSSION TITLE
MS Teams in web browser - config help joining video call

### DISCUSSION THREAD

**Discussion by db-rtaylor** [johnste/finicky#239] [General]
I'm on an Intel Mac & MS Teams is a process hungry POS, so have set up Teams in [MS Edge as a PWA](https://luiyongsheng.medium.com/native-microsoft-teams-performance-on-m1-macs-8d6483fa52e4) which is much friendlier on CPU.

This led me to finicky as a way of opening the Edge PWA when a teams meeting link is clicked. This works to a point with my config (attached (https://github.com/johnste/finicky/files/7946402/finicky.txt)) which I only set up earlier this week.

The PWA opens when a meeting invite is clicked, but it doesn't open into the call window but in the "Apps" window. From there I have to click "calendar" and then click to join the meeting. Does anyone have an idea of how I could change the config to follow the click directly to a meeting? (This works in a standard browser when clicking a meeting URL without finicky hijacking the click and directing to the PWA).

Here's an example of what the the URL in Outlook (feel free to click it's a dummy meeting I've created)

{ HYPERLINK "https://teams.microsoft.com/l/meetup-join/19%3ameeting_MmE1MTU5MWItMDZhNC00YWM0LTllOWUtOTVlM2I0NTk2OTYz%40thread.v2/0?context=%7b%22Tid%22%3a%2211f603b0-48ee-4ebe-99ef-47de1d5be024%22%2c%22Oid%22%3a%229421ee56-9012-45be-a75a-8b9328e5378c%22%7d" \t "_blank" }

And the URL the same link resolves to if you allow it to open in a standard browser https://teams.microsoft.com/_#/pre-join-calling/19:meeting_MmE1MTU5MWItMDZhNC00YWM0LTllOWUtOTVlM2I0NTk2OTYz@thread.v2

Any assistance getting this running would be greatly appreciated!

**Comment by sepsamavi** [comment]
See: https://github.com/johnste/finicky/wiki/Configuration-ideas
